bokeh/server/server.py

```diff
@@ -471,7 +479,7 @@ def address(self) -> str | None:
 # documentation elsewhere)
 class _ServerOpts(Options):
 
-    num_procs = Int(default=1, help="""
+    num_procs: int = Int(default=1, help="""
```

Comment:
This is a trial balloon for one possible way to type properties. If we can develop a better alternative that is not monstrous to implement, then that would be great. But all the alternatives I've seen seem like they would be incredibly invasive (unlike this, which would merely be incredibly tedious, once)